src/Appwrite/Platform/Workers/Databases.php

```diff
@@ -1,76 +1,94 @@
 <?php
 
+namespace Appwrite\Platform\Workers;
+
 use Appwrite\Event\Event;
-use Appwrite\Extend\Exception;
 use Appwrite\Messaging\Adapter\Realtime;
-use Appwrite\Resque\Worker;
+use Exception;
```

Comment:
Shouldn't we be using \Exception when we need this?

Reply:
Why not import it instead?